geopyspark/geotrellis/layer.py

```diff
@@ -474,39 +472,31 @@ def collect_metadata(self, layout=LocalLayout()):
 
         return Metadata.from_dict(json.loads(json_metadata))
 
-    def reproject(self, target_crs, layout=None, resample_method=ResampleMethod.NEAREST_NEIGHBOR):
+    def reproject(self, target_crs, resample_method=ResampleMethod.NEAREST_NEIGHBOR):
         """Reproject rasters to ``target_crs``.
-        When layout is ``None``, the reproject does not sample past tile boundary.
-        When layout is given the reproject will tile the rasters and produce a tiled layer.
+        The reproject does not sample past tile boundary.
 
         Args:
             target_crs (str or int): Target CRS of reprojection.
                 Either EPSG code, well-known name, or a PROJ.4 string.
-            layout (
-                :obj:`~geopyspark.geotrellis.GlobalLayout` or
-                :class:`~geopyspark.geotrellis.LocalLayout`, optional
-            ): Target raster layout when reprojecting.
             resample_method (str or :class:`~geopyspark.geotrellis.constants.ResampleMethod`, optional):
                 The resample method to use for the reprojection. If none is specified, then
                 ``ResampleMethods.NEAREST_NEIGHBOR`` is used.
 
         Returns:
-            :class:`~geopyspark.geotrellis.rdd.RasterLayer` or
-            :class:`~geopyspark.geotrellis.rdd.TiledRasterLayer`
+            :class:`~geopyspark.geotrellis.rdd.RasterLayer`
         """
 
         resample_method = ResampleMethod(resample_method)
 
         if isinstance(target_crs, int):
             target_crs = str(target_crs)
 
-        if layout is None:
-            srdd = self.srdd.reproject(target_crs, resample_method)
-            return RasterLayer(self.pysc, self.layer_type, srdd)
-        else:
-            return _reproject(target_crs, layout, resample_method, self)
+        srdd = self.srdd.reproject(target_crs, resample_method)
 
-    def tile_to_layout(self, layout=LocalLayout(), resample_method=ResampleMethod.NEAREST_NEIGHBOR):
+        return RasterLayer(self.pysc, self.layer_type, srdd)
+
+    def tile_to_layout(self, layout=LocalLayout(), target_crs=None, resample_method=ResampleMethod.NEAREST_NEIGHBOR):
```

Comment:
👍 I like `target_crs` here over just `crs`, its helpful in clearing up the intent of the parameter.